app/blueprints/fund_builder/forms/round.py

```diff
@@ -17,10 +17,10 @@ class RoundForm(FlaskForm):
         description="Choose a unique short name with 6 or fewer characters",
         validators=[DataRequired(), Length(max=6)],
     )
-    opens = DateTimeField("Opens")
-    deadline = DateTimeField("Deadline")
-    assessment_start = DateTimeField("Assessment Start")
-    reminder_date = DateTimeField("Reminder Date")
-    assessment_deadline = DateTimeField("Assessment Deadline")
+    opens = DateTimeField("Opens", format="%d-%m-%Y %H:%M", validators=[DataRequired()])
```

Comment:
I noticed when it loads the data back into the fund form (eg. when it fails validation, like you enter one date but not another, so it tells you you have missed something and shows what you previously entered) it doesn't display it in this format, so you have to retype it. Please can you update the display format to match?